File: versioned_docs/version-1.9.x/guides/examples/py/src/data_replication_remove.py
import asyncio

from reduct import Client, ReplicationSettings


async def main():
    # Create a client instance
    client = Client("http://127.0.0.1:8383", api_token="my-token")
    # Remove the `repl-to-remove` replication
    await client.delete_replication("repl-to-remove")


loop = asyncio.get_event_loop()
loop.run_until_complete(main())

File: versioned_docs/version-1.20.x/examples/rs/examples/data_replication_remove.rs
use reduct_rs::{ReductClient, ReductError};
use tokio;

#[tokio::main]
async fn main() -> Result<(), ReductError> {
    // Create a client instance
    let client = ReductClient::builder()
        .url("http://127.0.0.1:8383")
        .api_token("my-token")
        .build();

    // Remove the `repl-to-remove` replication
    client.delete_replication("repl-to-remove").await?;
    Ok(())
}
